### PR TITLE
Enable the `avc` check for all `tmt` tests

### PR DESCRIPTION
**Pull request by psss**
Let's check for AVC denials when initiated by Packit.

Pull Request Checklist

* [x] extend the test coverage

### CODE REVIEW

File: tests/main.fmf [hunk 1]
```diff
@@ -4,3 +4,7 @@ contact: Petr Šplíchal <psplicha@redhat.com>
 tier: 2
 tag: [container, virtual]
 require: [tmt]
+
+adjust:
+  - check: avc
```
Comment (happz) [inline tests/main.fmf:9]: I'd recommend making `check` a list right away. There might be tests that will add more checks in the future, and it's ugly to decode "cannot add list and dict" error message spitted out by fmf :/

Comment (psss) [reply tests/main.fmf]: Makes sense, fixed in 06da5fda.

**Review by happz — APPROVED** [review]

**Pull request merged**